physical/raft/msgpack.go

```diff
@@ -0,0 +1,11 @@
+package raft
+
+import (
+	"github.com/hashicorp/go-msgpack/codec"
```

Comment:
Does it complain if you change it to something like this without referencing anything from the package?
```suggestion
	_ "github.com/hashicorp/go-msgpack/codec"
```

Reply:
Nope, good idea.